From a93b55e9e0dc659c457d943c9520d18191e2e6f0 Mon Sep 17 00:00:00 2001
From: Debarshi Ray <debarshir@gnome.org>
Date: Tue, 29 May 2018 18:58:40 +0200
Subject: [PATCH 2/4] meson: Unbreak the non-macOS builds

The name_suffix keyword argument identifies an empty array, not an
empty string, as None or null [1].  Using an empty string creates
shared objects without any suffix whatsoever on non-macOS
environments, which GModule can't load.

Fallout from 8d3367c2c500a8c9f67f0447c4f711ca89625ae4

[1] As per the implementation of the BuildTarget class in
meson/mesonbuild/build.py

https://gitlab.gnome.org/GNOME/glib-networking/issues/33
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 1c52cc0..c875331 100644
--- a/meson.build
+++ b/meson.build
@@ -101,7 +101,7 @@ subdir('po')
 enable_installed_tests = get_option('installed_tests')
 test_template = files('template.test.in')
 
-module_suffix = ''
+module_suffix = []
 # Keep the autotools convention for shared module suffix because GModule
 # depends on it: https://gitlab.gnome.org/GNOME/glib/issues/520
 if ['darwin', 'ios'].contains(host_system)
-- 
2.17.0

